server/tests/mochData/socialMock.js

```diff
@@ -0,0 +1,63 @@
+import faker from 'faker';
+
+const id = faker.random.uuid();
+const firstName = faker.name.firstName();
+const lastName = faker.name.lastName();
+const email = faker.internet.email();
+
+const googleProfile = {
+  provider: 'google',
+  id,
+  displayName: `${firstName} ${lastName}`,
+  name: { familyName: lastName, givenName: firstName },
+  emails: [{ value: email, type: 'ACCOUNT' }],
+  photos: [
+    {
+      value: 'https://lh3.googleusercontent.com/a-/AAuE7mD7hIQTg4_0WvV34It93y5tEeDT87JbKRW5Aln_=s50'
```

Comment:
Put a faje value here as well. No need to provide real one. You can replace the content from **.com/** to any invalid url/data.

Reply:
Feedback Implemented!